recipes/orgalorg/bld.bat

```diff
@@ -0,0 +1,13 @@
+go build -mod=mod -buildmode=pie -trimpath -o=%LIBRARY_PREFIX%\bin\%PKG_NAME%.exe -ldflags="-s -w -X main.version=%PKG_VERSION%" || goto :error
+go-licenses save . --save_path=license-files ^
+	--ignore github.com/kovetskiy/lorg ^
```

Comment:
Are these packages without a license or is only go-licenses failing on them? If the latter, please add them manually.